addons/l10n_vn_edi/__manifest__.py

```diff
@@ -0,0 +1,30 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+{
+    "name": """Vietnam - E-invoicing""",
+    'icon': '/account/static/description/l10n.png',
+    "version": "1.0",
+    'countries': ['vn'],
+    "category": "Accounting/Localizations/EDI",
+    'website': 'https://www.odoo.com/documentation/master/applications/finance/fiscal_localizations/vietnam.html',
+    "depends": [
+        "l10n_vn",
+    ],
+    "summary": "E-invoicing using SInvoice by Viettel",
+    "description": """
+Vietnam - E-invoicing
+=====================
+Using SInvoice by Viettel
```

Comment:
It could, I checked and saw that for other countries we didn't add this information in the name.
That's why I added it in the summary (so that searches are easier).

If you ask to add it, there, I'm ok with that.